### DISCUSSION TITLE
Suggested: EVIL

### DISCUSSION THREAD

**Discussion by LeoBadeaux** [Prodigy-Hacking/ProdigyMathGameHacking#967] [(A) Anything]
![image](https://user-images.githubusercontent.com/79180322/118654581-b4316900-b7b6-11eb-92ce-82bd076161d3.png)
Github thinks we're evil now

**Comment by NX721** [comment]
Lol

**Comment by ArcerionDev** [comment]
I removed it and it's now suggesting "risk"
![image](https://user-images.githubusercontent.com/60945964/118689194-bc989c80-b7d4-11eb-9576-e1291b22fc7a.png)

**Comment by ghost** [comment]
Why evil they are bullying everyone here

*jk lol*
